styles/components/_search.scss

```diff
@@ -203,6 +206,33 @@
     padding: 0px 32px;
   }
 
+  .image-outline {
+    border: 1px solid $black;
+    border-radius: 50%;
+  }
+
+  .search-history {
+    margin-top: 20px;
+    color: $k-grey;
+    cursor: pointer;
+
+    .history-label {
+      font-size: 16px;
+    }
+
+    .times-icon {
+      &:hover {
+        color: $black;
+      }
+    }
+
+    .search-hisotry-item {
```

Comment:
@prachi00 you can fix it in supplemental PR please 🙏

Reply:
could be fixed in another PR I guess

Reply:
I'll fix this yes